### PR TITLE
[ACTION] Refactor Bitbucket actions #1484

### PR DESCRIPTION
**Pull request by lcaresia**

### CODE REVIEW

File: components/bitbucket/package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
     "name": "@pipedream/bitbucket",
-    "version": "0.3.3",
+    "version": "0.3.4",
     "description": "Pipedream Bitbucket Components",
     "main": "bitbucket.app.js",
```
Comment (jcortes) [inline components/bitbucket/package.json:5]:
```suggestion
    "main": "bitbucket.app.mjs",
```

**Review by jcortes — CHANGES_REQUESTED** [review]
Hi @lcaresia I've just added a few minor request changes. Everything else looks good to me. Thanks!

File: components/bitbucket/actions/common/common.mjs [hunk 2]
```diff
@@ -0,0 +1,13 @@
+import bitbucket from "../../bitbucket.app.mjs";
```
[the remark this answers is not in the document]
Comment (lcaresia) [reply components/bitbucket/actions/common/common.mjs:1]: What the difference between `common` and `base` in this case ?

This is documented in some place?

File: components/bitbucket/package.json [hunk 3]
```diff
@@ -10,9 +10,7 @@
     "homepage": "https://pipedream.com/apps/bitbucket",
     "author": "Pipedream <support@pipedream.com> (https://pipedream.com/)",
     "license": "MIT",
-    "dependencies": {
-        "axios": "^0.21.1"
-    },
+    "dependencies": {},
```
[the remark this answers is not in the document]
Comment (lcaresia) [reply components/bitbucket/package.json:13]: Why ? Now is used the `@pipedream/platform` axios

[the remark this answers is not in the document]
Comment (jcortes) [reply components/bitbucket/package.json:13]: Yeah I meant to say `@pipedream/platform`. From now on please add all third party dependencies in `package.json`

**Review by jcortes — DISMISSED** [review]
@lcaresia LGTM! Ready for QA!

**Comment by DilanAthukorala** [comment]
@lcaresia 

Getting following error for get_file_from_repository 

image


```
<?xml version="1.0" encoding="UTF-8"?>
<Error><Code>InvalidArgument</Code><Message>Only one auth mechanism allowed; only the X-Amz-Algorithm query parameter, Signature query string parameter or the Authorization header should be specified</Message><ArgumentName>Authorization</ArgumentName><ArgumentValue>Bearer uVSxvDHdzlNH2E7dzJDvhfTNN6pWdfQtdnHhbhOfct7egoIk0SAi5oPqdRpAcpXjT9b_WgmypMavmfM9oliOVj5AoGk6nwZjSKQaFXvLaXCAQuZKOpsrw5o7</ArgumentValue><RequestId>0D07P410RDA0D8PX</RequestId><HostId>Ucow3AGZjSlY5qgFbCb5q30C/X+GxvTrEnMueUva+NvdeEZ54PYQQzOXhcY5NcDd5Let7s6rrcM=</HostId></Error>
```

**Comment by lcaresia** [comment]
@DilanAthukorala Can you take a look now? I made some updates.

**Comment by DilanAthukorala** [comment]
Verification successful no issues observed.

**Review by michelle0927 — APPROVED** [review]

**Pull request merged**